Source/WebCore/page/PointerLockController.h

```diff
@@ -27,6 +27,7 @@
 #if ENABLE(POINTER_LOCK)
 
 #include "ExceptionCode.h"
+#include "Page.h"
```

Comment:
I don't think we should include `Page.h` if we're moving the implementation of ref/deref to cpp file.